classes/local/object_manipulator/candidates/candidates_factory.php

```diff
@@ -33,6 +33,9 @@
 use tool_objectfs\local\object_manipulator\recoverer;
 use tool_objectfs\local\object_manipulator\orphaner;
 
+/**
+ * Candiates Factory
```

Comment:
is this a factory where candy is eaten or just a typo? :-)

Reply:
Just a typo :)